src/main/fc/fc_init.c

```diff
@@ -367,6 +364,11 @@ void init(void)
 #endif /* USE_SPI */
 
 #ifdef USE_I2C
+    i2cHardwareConfigure(i2cConfig());
```

Comment:
Is there any advantage passing pointer to config? PGs are usually used internally ...